types/ocpp.yaml

```diff
@@ -633,3 +633,32 @@ types:
         description: Detailed status information.
         type: object
         $ref: /ocpp#/StatusInfoType
+  RegistrationStatus:
+    description: Indicates whether the Charging Station has been registered within the OCPP CSMS
+    type: string
+    enum:
+      - Accepted
+      - Pending
+      - Rejected
+  BootNotificationResponse:
+    description: OCPP BootNotificationResponse
+    type: object
+    required:
+      - status
+      - current_time
+      - interval
+    properties:
+      status:
```

Comment:
For 2.0.1 there is also a optional "statusInfo" of StatusInfoType containing a "reason code" and "additional info" (both strings). This could also be added to this type, but would not be available for 1.6 anyway